suite/soak/ocr_test_deploy.go

```diff
@@ -0,0 +1,41 @@
+package soak_runner
+
+//revive:disable:dot-imports
+import (
+	"fmt"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink-testing-framework/actions"
+	"github.com/smartcontractkit/chainlink-testing-framework/config"
+	"github.com/smartcontractkit/chainlink-testing-framework/utils"
+	"github.com/smartcontractkit/helmenv/environment"
+)
+
+var _ = Describe("OCR Soak Setup @setup-soak", func() {
+	It("Deploys soak test runner", func() {
+		actions.LoadConfigs()
+		exePath := buildGoTests()
+
+		env, err := environment.DeployRemoteRunnerEnvironment(
+			environment.NewChainlinkConfig(
+				environment.ChainlinkReplicas(6, config.ChainlinkVals()),
+				"zchainlink-soak",
```

Comment:
Do you still want this named "zchainlink-soak", maybe something like "chainlink-ocr-soak"?

Reply:
I forgot to flag these, but I agree with both things Tate said.